src/components/lib/posix/posix.c

```diff
@@ -457,12 +477,12 @@ pre_syscall_default_setup()
 {
 	printc("pre_syscall_default_setup\n");
 
-	struct cos_defcompinfo *defci = cos_defcompinfo_curr_get();
-	struct cos_compinfo    *ci    = cos_compinfo_get(defci);
+	// struct cos_defcompinfo *defci = cos_defcompinfo_curr_get();
```

Comment:
So this function is basically doing nothing. Can we remove it? Is there a reason to keep it?

Reply:
This is the protocol when starting libposix program. We might want add other initialization steps here.